### DISCUSSION TITLE
Redlining/Umgebungskreis - Anwenderwunsch

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#56] [General]
Bitte die Beschriftungstexte etwas deutlicher hervorheben, damit sie am Ausdruck (PDF) noch besser lesbar sind. Der Wunsch kommt von unseren Lärmschutz-Sachverständigen.

**Comment by jugstalt** [comment]
Hast hu hier ein Beispiel, was "deutlicher hervorheben" bedeutet. Bei mir ist das im Ausdruck auch mit Luftbild im Hintergrund gut lesbar 150 dpi:
![image](https://user-images.githubusercontent.com/26577522/232502919-35969c20-0be2-49a1-b63d-ab21bbcb8d74.png)

Selbst mit 120 DPI ist die Schrift vom Umgebungskreis noch besser lesbar, als die Grundstücksnummer von AGS:
![image](https://user-images.githubusercontent.com/26577522/232503479-ae5effd4-8590-49e1-a843-e50b4c3ea900.png)


Mit wie viel DPI druckt ihr aus.?

**Reply by cmairamhof** [reply]
Jetzt habe ich ein Beispiel, A4 Hochformat, M 1:5000, Qualität 288 dpi.  Jedenfalls skaliert der Text bei hoher Druckauflösung nicht mit. Die Skalierung musss aber nicht sein - eventuell einfach den Text 25% größer machen?

![image](https://user-images.githubusercontent.com/79247887/232508903-9e0c4f45-2eaa-4948-903a-0bf5c079e51f.png)

**Comment by gis-eni** [comment]
Ah, es skaliert nicht mit der Auflösung. Diesen "Bug" konnte ich beheben.
Sollte im nächsten Build passen
